DOCS-3213: delete API folder and contents (#2083)
MultiSafepay/docs: content/integration/community/Community integration requirements/_index.md

If you want to publish a MultiSafepay community integration, you have to adhere to the following agreements:

> * The plugin information should be visible in the JSON format and include the configuration of the integration. See [Create order](https://docs-api.multisafepay.com/reference/createorder) > Body params > .. > Plugin for more detail.

Reviewer: ```suggestion
* The plugin information should be visible in the JSON format and include how to configure the integration. For more information, see [Create order](https://docs-api.multisafepay.com/reference/createorder) > Body params > .. > Plugin.
```
